packages/clipper/src/index.ts

```diff
@@ -476,6 +477,29 @@ async function getPage(
   title.innerText = document.title;
   head.appendChild(title);
 
+  if (config?.styles) {
+    for (const sheet of document.styleSheets) {
+      const node = sheet.ownerNode;
+      const href =
+        sheet.href && node instanceof HTMLLinkElement
+          ? node.href
+          : node instanceof HTMLStyleElement
+          ? node.getAttribute("href")
+          : null;
+      if (href) {
+        const styleNode = await downloadStylesheet(
+          href,
+          resolveFetchOptions(config)
+        );
+        if (styleNode) {
+          head.appendChild(styleNode);
+        }
+      } else if (node instanceof HTMLStyleElement) {
+        head.appendChild(node.cloneNode(true));
+      }
+    }
+  }
```

Comment:
@thecodrr the `getInlineNode` function (which calls `inlineStylesheets`) doesn't create/operate on the `head` at all. The `head` element is created here, so the inlining style logic is still here. Do let me know what would be the correct way to handle this

Reply:
We can move this into a function in `styles.ts` and call it here.